pyproject.toml

```diff
@@ -85,6 +75,16 @@ codemod = [
     "ruff==0.12.11",
     "autotyping==23.2.0",
 ]
+docs = [
+    "sphinx==8.2.3; python_version >= '3.11'",
+    "sphinx-autobuild~=2025.8.25 ; python_version >= '3.11'",
+    "sphinx-hoverxref==1.4.2; python_version >= '3.11'",
+    "sphinx-notfound-page==1.1.0; python_version >= '3.11'",
+    "sphinxcontrib-towncrier==0.5.0a0; python_version >= '3.11'",
+    "sphinxcontrib-trio~=1.1.2; python_version >= '3.11'",
+    "sphinxext-opengraph==0.9.1",
+    { include-group = "changelog" },
```

Comment:
apparently pyright's parser wasn't fixed for this until 1.1.387, so bumping pyright is necessary to dedupe dependencies.